eagleEvents/routes/customers.py

```diff
@@ -8,8 +10,8 @@
 @customers_blueprint.route('/listCustomers')
 @multi_auth.login_required
 def list_customers():
-    # TODO List customers
-    return render_template('customer.html.j2')
+    customers = Customer.query.all();
```

Comment:
Need to filter by current user's company since we can have multiple companies in the system with customers.  May need this change on all the list pages, just now thought about it after I already reviewed and merged the other two